### ISSUE TITLE
[REPO CREATION] Create container-device-interface-rs in cncf-tags

### ISSUE THREAD

**Issue by zvonkok** [cncf/toc#1389]
### Requested name for new repository

container-device-interface-rs

### What should the repo description be?

Rust implementation of the Container Device Interface: https://github.com/cncf-tags/container-device-interface

### What TAG/WG does this fall under?

TAG runtime

### Additional context for request

https://github.com/cncf-tags/container-device-interface/issues/150
https://github.com/kata-containers/kata-containers/issues/10059

**Comment by elezar** [comment]
@zvonkok if I recall correctly, you have a repo under your GitHub user that you would like to migrate to this new repository. Is that correct?

**Comment by zvonkok** [comment]
Correct, https://github.com/zvonkok/container-device-interface-rs/tree/main

**Comment by riaankleinhans** [comment]
@zvonkok can you please "transfer" the repo to cncf-tags in the repo setting.
I should see the notice of the transfer come through, but please let me know when it is done.

**Comment by elezar** [comment]
Is there anything else that is required here?
